docs/astro.config.mjs

```diff
@@ -9,6 +9,7 @@ export const locales = {
 	fr: { label: 'Français', lang: 'fr' },
 	it: { label: 'Italiano', lang: 'it' },
 	zh: { label: '简体中文', lang: 'zh' },
+	'pt-br': { label: 'Português do Brasil', lang: 'pt'}
```

Comment:
We use `pt-BR` in the lang tag in docs. Should that be the same here? Or do we just use `pt` and also drop the region specifier in the `label`?

Reply:
Oh, that's right, technically the correct lang is `pt-BR` but our links in docs are `pt-br`, so I wasn't sure which one to use. I have no strong feelings about changing to just `pt`, though I'd still keep the label as "Brazilian Portuguese" for disambiguation, does that seem like a bad idea?

Reply:
Personally I prefer the specific version, mainly because in the past it was more common to find `pt-PT` translations when it was just `pt` so seing that gives me bad flashbacks, but I think nowadays it has changed. Google Translate, for example, does not distinguish between the two and there is only the `pt` language (with "Portuguese" label), and as far as I know, it is the `pt-BR` version for the most part. Idk what portuguese people think about that, there is already that debate around portugueses starting to speak only `pt-BR` and "losing" their native language, but its clear that the brazilian community has much more presence now than in the past.

Reply:
I'll let you make the final call Yan, but just to clarify what the config does:

```
// 👇 Used as this language's directory and URL path — should be lowercase but doesn't *have* to be a correct BCP-47 tag
'pt-br': {
  // 👇 Used as the HTML Lang attribute for this language — should be a valid BCP-47 tag
  lang: 'pt-BR'
}
```
My only research was to see that Wikipedia only as a `pt` subdomain, uses `pt` as the `lang` tag and labels as "Português".

Reply:
Reading [this](https://meta.m.wikimedia.org/wiki/Requests_for_new_languages/Wikipedia_Brazilian_Portuguese) made me think about it, but idk the reasoning about `zh` and `zh-cn`